governance/lead-proposal-process.md

```diff
@@ -0,0 +1,153 @@
+2. If a clear successor for the exiting Co-Chair exists based on the merit of their contributions ("plant trees, carry water") and relevant experience that will benefit the TAG, based on the TAG's collective experience over time, the exiting Co-chair may nominate a successor to the TOC.
+3. The exiting Co-Chair should propose the desired successor to the other Co-Chairs. If there are any objections, the Co-Chair will have the opportunity to propose another worthy candidate. If Environmental Sustainability TAG Chairs, Tech Leads and TOC Liaisons are unable to put forward a candidate, then a selection process will need to be agreed upon and executed.
+4. If no new Chair candidate is available, the role will remain vacant until the Co-chair's, TOC Liaisons or CNCF TOC propose a replacement.
+
+### General nomination requirements
+
+The TAG ENV recognizes and values the achievements of individuals who have proven themselves in this community.
+If a given nominee does not meet all the requirements, the TAG leadership will still review the nomination and justification to determine if enough intent and commitment is present to move forward with an endorsement.
+
+**Nominees must**:
+
+* Be the author of at least one pull request against the TAG ENV repository
+* Be active in the community within the last 3 months before their nomination
+  * Active - Attending at least 1 TAG ENV Regular Meeting a month
+  * Engaging in community chat via Slack or lists
+  * Commenting on PRs and issues to drive suggestions to proposals, define scope, resolve clarity issues, etc.
+* Agree to the nomination
+
+*additional requirements may be defined in the specific [roles](roles.md) governance document*
+
+**Nominations must**:
+
+* Come from someone other than the nominee
+* Have justification from the nominator regarding the nominee's capabilities
+
+#### Justification for nominees
+
+To ensure nominations are thoughtful and supported, in addition to the above requirements, a justification must enumerate on the nominee's:
+
+* TAG Community involvement
+* Leadership and collaboration skills
+* Willingness to learn and expand existing technical knowledge
+
+### Announcing nomination results
+
+The TAG leadership will release aggregate information about the nominees at the end of the nomination process. Aggregate information will not include who nominated someone.
+
+### Next Steps
+
+If the role under nomination requires TOC approval or vote, the TAG Co-chairs are responsible for coordinating and executing that in accordance with the TOC's processes (defined previously).
+
+## Working Group and Project Leadership
+
+Working group and project leads are appointed by the TAG Chairs through the following process.
+
+### Process of nominations
+
+TAG Lead roles are open to all interested participants within the community. 
+There are currently no limits of leadership roles for working groups or projects if not specifed otherwise in the WG / project charter document. 
```

Comment:
```suggestion
There are currently no limits of leadership roles for working groups or projects if not specifed otherwise in the WG / project charter document.
```
Markdown build error